modules/rostests/apitests/user32/CopyImage.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     Test for SetFocus/GetFocus/GetGUIThreadInfo
+ * COPYRIGHT:   Copyright 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#include "precomp.h"
+#include <versionhelpers.h>
+
+#define COPYIMAGE_VALID_FLAGS ( \
+    LR_SHARED | LR_COPYFROMRESOURCE | LR_CREATEDIBSECTION | LR_LOADMAP3DCOLORS | 0x800 | \
+    LR_VGACOLOR | LR_LOADREALSIZE | LR_DEFAULTSIZE | LR_LOADTRANSPARENT | LR_LOADFROMFILE | \
+    LR_COPYDELETEORG | LR_COPYRETURNORG | LR_COLOR | LR_MONOCHROME \
+)
+
+static HANDLE CreateTestImage(UINT uType)
+{
+    HANDLE hImage;
+    HDC hDC;
+    switch (uType)
+    {
+        case IMAGE_BITMAP:
+            hDC = CreateCompatibleDC(NULL);
+            hImage = (HANDLE)CreateCompatibleBitmap(hDC, 10, 10);
+            DeleteDC(hDC);
+            break;
+        case IMAGE_CURSOR:
+            hImage = (HANDLE)LoadCursor(NULL, IDC_ARROW);
+            break;
+        case IMAGE_ICON:
+            hImage = (HANDLE)LoadIcon(NULL, IDI_APPLICATION);
+            break;
+    }
+    return hImage;
+}
+
+static VOID
+Test_CopyImage_Flags(UINT uType)
+{
+    UINT iBit, uBit, uValidFlags = COPYIMAGE_VALID_FLAGS;
+    HANDLE hImage = CreateTestImage(uType), hCopyedImage;
+
+    if (IsWindowsVistaOrGreater())
+        uValidFlags |= 0x10000;
```

Comment:
What's this flag? Where did you find it?

Reply:
I found it in My Win10 PC.

Reply:
Obviously the question was, _how_ did you find it, by doing which tests?

Reply:
By tests.